deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import socket
+
+from wazuh_testing.tools import ARCHIVES_LOG_FILE_PATH
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor, make_callback, REMOTED_DETECTOR_PREFIX
+
+
+def callback_detect_remoted_started(port, protocol, connection_type="secure"):
+    """Creates a callback to detect if remoted was correctly started
+
+    wazuh-remoted logs if it has correctly started for each connection type, the port and
+    the protocol in the ossec.log
+
+    Args:
+        port (int): port configured for wazuh-remoted.
+        protocol (str): protocol configured for wazuh-remoted. It can be UDP, TCP or both options at the same time.
+        connection_type (str): it can be secure or syslog.
+
+    Returns:
+        callable: callback to detect this event
+    """
+    msg = fr"Started \(pid: \d+\). Listening on port {port}\/{protocol} \({connection_type}\)."
+
+    return make_callback(pattern=msg, prefix=REMOTED_DETECTOR_PREFIX)
+
+
+def callback_detect_syslog_event(message):
+    """Creates a callback to detect the syslog messages in the archives.log
+
+    Args:
+        message (str): syslog message sent through the socket
+
+    Returns:
+        callable: callback to detect this event
+    """
+    expr = fr".*->\d+\.\d+\.\d+\.\d+\s{message}"
+    return make_callback(pattern=expr, prefix=None)
+
+
+def send_syslog_message(message, port, protocol, manager_address="127.0.0.1"):
+    """This function sends the ping message to the manager
```

Comment:
Change this, I think that was copy-pasted from `send_ping_pong_messages`

Reply:
Touché. Done in 3f381f0add7acadf3c21a21d945322ac62541e4e